packages/fiori/src/WizardPopover.hbs

```diff
@@ -1,6 +1,9 @@
 <ui5-responsive-popover
 	horizontal-align="Center"
 	placement-type="Bottom"
+	role="dialog"
+	aria-modal="true"
+	aria-labelledby="{{_id}}-actions {{_id}}-steps"
```

Comment:
do we have support for aria-labelledby?

I only see support for aria-label https://github.com/SAP/ui5-webcomponents/pull/1898/